pkg/machine/qemu/machine.go

```diff
@@ -402,6 +402,14 @@ func (v *MachineVM) Set(_ string, opts machine.SetOptions) ([]error, error) {
 		}
 	}
 
+	if opts.USBs != nil {
```

Comment:
In the case of `opts.USBs=""`, ie `podman machine set --usb=""`, will the usb be removed from the machine? If so, let's document this.

Reply:
No, that would be considered invalid input with the code we have now. I'll change to your suggestion plus the documentation.

Another option would be another command line argument that states cleaning/reset/disable the usb configuration, eg: `podman machine set --disable-usb`. But only usb would need it, so I'm not sure it is worth to introduce a new command line option, so I agree with `--usb=""` instead.

Reply:
This is enough to fix it. Thanks for the suggestion.

```diff
--- a/pkg/machine/qemu/config.go
+++ b/pkg/machine/qemu/config.go
@@ -66,6 +66,11 @@ func (v *MachineVM) setNewMachineCMD(qemuBinary string, cmdOpts *setNewMachineCM
 func parseUSBs(usbs []string) ([]machine.USBConfig, error) {
        configs := []machine.USBConfig{}
        for _, str := range usbs {
+               if str == "" {
+                       // Ignore --usb="" as it can be used to reset USBConfigs
+                       continue
+               }
+
                vals := strings.Split(str, ",")
                if len(vals) != 2 {
                        return configs, fmt.Errorf("usb: fail to parse: missing ',': %s", str)
```